python/ray/data/tests/test_mars.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+import ray
+import mars
+import mars.dataframe as md
+
+
+@pytest.fixture(scope="module")
+def ray_start_regular(request):  # pragma: no cover
+    try:
+        yield ray.init(num_cpus=16)
+    finally:
+        ray.shutdown()
+
+
+def test_mars(ray_start_regular):
+    import pandas as pd
+
+    cluster = mars.new_cluster_in_ray(worker_num=2, worker_cpu=1)
+    n = 10000
+    pdf = pd.DataFrame({"a": list(range(n)), "b": list(range(n, 2 * n))})
+    df = md.DataFrame(pdf)
+
+    # Convert mars dataframe to ray dataset
+    ds = ray.data.from_mars(df)
+    pd.testing.assert_frame_equal(ds.to_pandas(), df.to_pandas())
+    ds2 = ds.filter(lambda row: row["a"] % 2 == 0)
+    assert ds2.take(5) == [{"a": 2 * i, "b": n + 2 * i} for i in range(5)]
+
+    # Convert ray dataset to mars dataframe
+    df2 = ds2.to_mars()
```

Comment:
For the sake of complete coverage of `.to_mars()`, could you add test cases for (1) converting an Arrow dataset (i.e. a dataset with Arrow, not Pandas, blocks), and (2) converting a simple dataset (i.e. a dataset with simple blocks, like `ray.data.range(n)`) and confirm that it raises a `NotImplementedError` exception.

Reply:
I added tests for arrow datasets. I didn't add tests for datasets like `ray.data.range(n)` since its not datafram compatible. I think we can convert such datasets into mars tensor in future.

Reply:
For simple datasets, I was suggesting that you add a test that confirms that the `NotImplementedError` exception is raised, but since that's rather nit-picky, this LGTM!

Reply:
My mistake, I misunderstood your suggesion. Just added test for a simple dataset.